pymeasure/instruments/hp/hp8116a.py

```diff
@@ -35,6 +35,30 @@
 log.addHandler(logging.NullHandler())
 
 
+def _generate_1_2_5_sequence(min, max):
+    """ Generate a list of a 1-2-5 sequence between min and max. """
+    exp_min = int(np.log10(min))
+    exp_max = int(np.log10(max))
+
+    seq_1_2_5 = np.array([1, 2, 5])
+    sequence = np.array([seq_1_2_5 * (10 ** exp) for exp in range(exp_min - 1, exp_max + 1)])
+    sequence = sequence.flatten()
+    sequence = sequence[(sequence >= min) & (sequence <= max)]
+
+    return list(sequence)
+
+
+def _boolean_control(identifier, state_index, docs, inverted=False, **kwargs):
+    return Instrument.control(
+        'CST', identifier + '%d', docs,
+        validator=strict_discrete_set,
+        values=[True, False],
+        get_process=lambda x: inverted ^ bool(int(x[state_index][1])),
+        set_process=lambda x: int(inverted ^ x),
+        **kwargs
+    )
```

Comment:
Annoying behaviour in python<3.10 with these static-methods. Is this something we'd like to revert when drop support for pyton<=3.9 (so in about 1.5 year I think)? If so, we might want to mark this as something to change back by then.

Reply:
I added a comment.